### PR TITLE
gradle wrapper fix

### PR DESCRIPTION
**Pull request by travisspencer**
The `.gitignore` file caused the `gradle-wrapper.jar` to be ignored and was thus not added to git's index. Fixed by regenerating and adding the JAR. The newer(?) version of gradle that I used also updated the wrapper scripts.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -28,3 +25,6 @@ hs_err_pid*
 # IntelliJ
 .idea/
 *.iml
+
+# Avoid ignoring Gradle wrapper jar file (.jar files are usually ignored)
+!gradle-wrapper.jar
```
Comment (renatoathaydes) [inline .gitignore:30]: I don't get it... looks like just the position was changed?

File: gradlew.bat [hunk 2]
```diff
@@ -60,11 +59,6 @@ set _SKIP=2
 if "x%~1" == "x" goto execute
 
 set CMD_LINE_ARGS=%*
-goto execute
-
-:4NT_args
-@rem Get arguments from the 4NT Shell from JP Software
-set CMD_LINE_ARGS=%$
```
Comment (renatoathaydes) [inline gradlew.bat:67]: Are these auto-generated changes by Gradle?

**Review by renatoathaydes — COMMENTED** [review]
This is my review summary :)

Looks good to me!

**Comment by renatoathaydes** [comment]
Merging because I just noticed I also can't run ./gradlew :(

File: .gitignore (hunk 1 above)
Comment (travisspencer) [reply .gitignore:30]: Ya, by having `*.jar` below `!gradle-wrapper.jar`, `graddle-wrapper.jar` was being ignored. By moving it below, all `*.jar` files except for `gradle-wrapper.jar` will be ignored.

File: gradlew.bat (hunk 2 above)
Comment (travisspencer) [reply gradlew.bat:67]: ya, this was done by running `gradle wrapper --gradle-version=2.13` again. I'm using Gradle 3.1, so I'm guessing a newer (or older?) version removed these things from the auto-generated scripts.

**Pull request merged**